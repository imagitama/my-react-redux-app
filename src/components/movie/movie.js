import React from 'react'
import styled from 'react-emotion'

const StyledMovie = styled('li')({
  listStyle: 'none',
  padding: '0.5rem',
  borderTop: '1px solid grey',
  ':first-child': {
    borderTop: 'none'
  }
})

// TODO: Connect directly to state using movie ID?
export const Movie = ({ title, releaseYear }) => 
  <StyledMovie>
    {title} ({releaseYear})
  </StyledMovie>