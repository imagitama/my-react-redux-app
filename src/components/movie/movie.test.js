import React from 'react'
import { mount } from 'enzyme'
import { Movie } from './movie'

it('Matches the snapshot', () => {
  expect(mount(<Movie title='My movie' releaseYear='2000' />)).toMatchSnapshot()
})

// TODO: Test if props are empty