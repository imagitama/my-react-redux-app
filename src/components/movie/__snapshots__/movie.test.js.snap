// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<Component
  releaseYear="2000"
  title="My movie"
>
  <Styled(li)>
    <li
      className="css-1eg3kgk"
    >
      My movie
       (
      2000
      )
    </li>
  </Styled(li)>
</Component>
`;
